cluster-autoscaler/cloudprovider/alicloud/examples/cluster-autoscaler-standard.yaml

```diff
@@ -63,8 +63,8 @@ rules:
   verbs: ["create"]
 - apiGroups: [""]
   resources: ["configmaps"]
-  resourceNames: ["cluster-autoscaler-status"]
-  verbs: ["delete","get","update"]
+  resourceNames: ["cluster-autoscaler-status", "cluster-autoscaler-priority-expander"]
```

Comment:
Once this merges we should also update https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/rbac/cluster-autoscaler/cluster-autoscaler-rbac.yaml#L54.